x-pack/platform/plugins/private/reporting/server/saved_objects/scheduled_report/schemas/v1.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { scheduleRruleSchema } from '@kbn/task-manager-plugin/server';
+
+const rawLayoutIdSchema = schema.oneOf([
+  schema.literal('preserve_layout'),
+  schema.literal('print'),
+  schema.literal('canvas'),
+]);
+
+export const rawNotificationSchema = schema.object({
+  email: schema.maybe(
+    schema.object({
+      to: schema.arrayOf(schema.string(), { minSize: 1 }),
```

Comment:
I wonder if someone will want "bcc only" or such.  I think in the email connector, we make sure the count of all these is at least one.  Certainly survivable for now, and "loosening" this in the future shouldn't cause any migration issues, if we want to do that ...

Reply:
Updated to require email in any of the fields: https://github.com/elastic/kibana/pull/219771/commits/55c7011cd023dabc5c4fea4b47f3d2db6a47fde6